pkg/resource/cluster.go

```diff
@@ -147,7 +149,10 @@ func (cluster Cluster) StatefulSetName() string {
 }
 
 func (cluster Cluster) JobName() string {
-	return fmt.Sprintf("%s-version-checker", cluster.Name())
```

Comment:
We need to eventually check to see if we are over the number of characters for a job name.  Adding the slug could take us over that amount.